### ISSUE TITLE
Change user list to preserve search string & paging

### ISSUE THREAD

**Issue by seankwilliams** [codefordenver/Comrad#871]
When using the page at `/user/search`, if you go to the second page or use the search term, and then go back, the paging and search filters is not preserved. You are back to page one, without a search string.

Ideally this page should preserve everything when you go back:
- paging
- Whether All, Active, or Inactive is preserved
- The value of the search box
- Scroll position

However, recognizing that the scroll position can be challenging to implement, if we could just get the paging, filters and search string preserved that would be a huge win, and we can make a separate issue to resolve the scroll position.

**Comment by kmid5280** [comment]
@seankwilliams I'm attempting to duplicate this error. It looks like the "previous" and "next" buttons under the search results work as intended.

When you say "go to the second page," do you mean clicking on one of the search results? Is this an issue that mainly happens when navigating away from the search page entirely?

**Comment by seankwilliams** [comment]
@kmid5280 You're right, I misphrased things in the original description. This is a problem when you go click on a row to go into a user page and then try to go back to the list.
